lib/matchers/matchers.rb

```diff
@@ -340,3 +340,16 @@ def format_expectation(negate)
     end
   end
 end
+
+# user resource belong_to matcher for serverspec compatibility
+RSpec::Matchers.define :belong_to_primary_group do |group|
+  match do |user|
+    user.belongs_to_primary_group?(group)
+  end
+end
+
+RSpec::Matchers.define :belong_to_group do |group|
+  match do |user|
+    user.belongs_to_group?(group)
+  end
+end
```

Comment:
In order to be able to support the `belong_to_group` and `belong_to_primary_group` matchers, I felt the need to define the custom matchers as `belong_to_` matcher didn't exist as part of InSpec or RSpec. 

Since other custom matchers were defined here, I thought this would be the right place to define them. Am I missing something here?

Reply:
Moved the custom matchers to `user.rb`